.chloggen/k8s_container_metrics.yaml

```diff
@@ -0,0 +1,27 @@
+# Use this changelog template to create an entry for release notes.
+
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: 'enhancement'
+
+# The name of the component, or a single word describing the area of concern, (e.g. filelogreceiver)
+component: kubeletstatsreceiver
+
+# A brief description of the change.  Surround your text with quotes ("") if it needs to start with a backtick (`).
+note: Add k8s.container.cpu.node_limit_utilization metric
```

Comment:
The existing metric names for conatiners/pods have `limit` in it to differentiate between the soft/hard limits (request vs limit) in k8s. In case of utilization wrt to k8s node, if we treat `capacity` as the only limit and do not think we'd want to add similar utilization metric computed based on the node's `allocatable` (capacity - kube/system reserved) resources, then `k8s.container.cpu.node_utilization` sounds good. But if we see capacity and allocatable as different limits for k8s nodes then we should add the limit type in the metric like we have request/limit. Semantic convention does not have any guidance on how to handle utilization when dealing with different limit types (soft vs hard) or limiting entities (node vs pod vs container), so taking this discussion to the conventions repo could be helpful.

Reply:
I wouldn't block this PR from getting merged waiting for a SemConv decision. We don't have any SemConv for k8s metrics at the moment. As part of https://github.com/open-telemetry/semantic-conventions/issues/1032 the k8s metric names will be revisited anyways, and even the `k8s.container.cpu_limit_utilization` will need to change to `k8s.container.cpu.limit_utilization` to follow the SemConv guidelines. 

So I believe that `k8s.container.cpu.node_utilization` follows the current SemConv guidelines.
However, if we plan to expand and introduce an additional metric to report against the node's `allocatable`, then I would go with something like `k8s.container.cpu.node.capacity_utilization` so as to preserve "space" for the `k8s.container.cpu.node.allocatable_utilization`.

I guess if we want to be future proof then `k8s.container.cpu.node.capacity_utilization` is our best option here?